ZitiMobilePacketTunnel/IdentityViewController.swift

```diff
@@ -588,7 +588,7 @@ class IdentityViewController: UITableViewController, MFMailComposeViewController
                 cell?.detailTextLabel?.text = zid?.name
             } else if indexPath.row == 1 {
                 cell?.textLabel?.text = "Network"
-                cell?.detailTextLabel?.text = zid?.czid?.ztAPI
+                cell?.detailTextLabel?.text = zid?.czid?.ztAPIs?.joined(separator: ", ") ?? zid?.czid?.ztAPI
```

Comment:
better to show the current (or first) controller in the short term (and the matching version below)?